sdk/keyvault/keyvault-certificates/src/certificatesModels.ts

```diff
@@ -17,7 +17,7 @@ export const LATEST_API_VERSION = "7.3";
 /**
  * The optional parameters accepted by the KeyVault's KeyClient
  */
-export interface CertificateClientOptions extends coreHttp.PipelineOptions {
+export interface CertificateClientOptions extends coreClient.CommonClientOptions {
```

Comment:
Isn't there supposed to be some ExtendedClientOptions or something along those lines? So sorry, I just haven't been following that effort too closely - where does core-http-compat fit into this?